src/components/DuplicatesPage/DuplicatesTable/DuplicateTableRow/DuplicateTableRow.js

```diff
@@ -0,0 +1,44 @@
+import SelectableTableCell from "../../../Common/SelectableTableCell/SelectableTableCell";
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import "./duplicateTableRow.less";
+import RoundSwitch from "../../../Common/RoundSwitch/RoundSwitch";
+import classNames from "classnames";
+import _ from "lodash";
+
+class DuplicateTableRow extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.onIncludeChange = this.onIncludeChange.bind(this);
+    }
+
```

Comment:
change bindings, do partial import, extract method from return to (map is too big) add defaultProps